x/leverage/keeper/token_test.go

```diff
@@ -27,3 +28,52 @@ func (s *IntegrationTestSuite) TestGetToken() {
 
 	require.Equal(uint32(24), t.HistoricMedians)
 }
+
+func (s *IntegrationTestSuite) TestTokenNoMigration() {
+	app, ctx, require := s.app, s.ctx, s.Require()
+
+	// Create an old token struct (had no HistoricMedians uint32 field)
+	oldToken := types.OldToken{
+		BaseDenom:              "uold",
+		SymbolDenom:            "OLD",
+		Exponent:               6,
+		ReserveFactor:          sdk.MustNewDecFromStr("0.2"),
+		CollateralWeight:       sdk.MustNewDecFromStr("0.25"),
+		LiquidationThreshold:   sdk.MustNewDecFromStr("0.25"),
+		BaseBorrowRate:         sdk.MustNewDecFromStr("0.02"),
+		KinkBorrowRate:         sdk.MustNewDecFromStr("0.22"),
+		MaxBorrowRate:          sdk.MustNewDecFromStr("1.52"),
+		KinkUtilization:        sdk.MustNewDecFromStr("0.8"),
+		LiquidationIncentive:   sdk.MustNewDecFromStr("0.1"),
+		EnableMsgSupply:        true,
+		EnableMsgBorrow:        true,
+		Blacklist:              false,
+		MaxCollateralShare:     sdk.MustNewDecFromStr("1"),
+		MaxSupplyUtilization:   sdk.MustNewDecFromStr("0.9"),
+		MinCollateralLiquidity: sdk.MustNewDecFromStr("0"),
+		MaxSupply:              sdk.NewInt(100_000_000000),
+	}
+
+	// Store the old token in state - simulating existing state
+	require.NoError(app.LeverageKeeper.SetOldToken(ctx, oldToken))
+
+	// Try to get it from state - using new token struct
+	t, err := app.LeverageKeeper.GetTokenSettings(ctx, "uold")
```

Comment:
Here's the meat of it. We can set and old token and get a new one no problem (since proto tolerates adding fields)